internal/manifests/queryfrontend/query_frontend.go

```diff
@@ -80,25 +83,31 @@ func BuildQueryFrontend(params manifestutils.Params) ([]client.Object, error) {
 		}
 	}
 
+	if tempo.Spec.Template.QueryFrontend.JaegerQuery.Enabled && tempo.Spec.Template.QueryFrontend.JaegerQuery.MonitorTab.Enabled &&
+		tempo.Spec.Template.QueryFrontend.JaegerQuery.MonitorTab.PrometheusEndpoint == thanosQuerierOpenShiftMonitoring {
```

Comment:
To make it easier for customers, should we set this value by default on OpenShift in the defaulter webhook, if the MonitorTab is enabled?

Reply:
Let's do it separately https://github.com/grafana/tempo-operator/issues/618